tests/unit/test_transaction.py

```diff
@@ -345,10 +354,23 @@ def test_commit_w_other_error(self):
 
         self.assertIsNone(transaction.committed)
 
+        if not HAS_OPENTELEMETRY_INSTALLED:
+            return
+
+        span_list = sorted(self.get_finished_spans(), key=lambda v: v.start_time)
+        got_span_names = [span.name for span in span_list]
+        want_span_names = [
+            "CloudSpanner.Transaction.commit",
+        ]
+        assert got_span_names == want_span_names
+
+        txn_commit_span = span_list[-1]
```

Comment:
same here, we are already asserting on span attributes below which is doing the same thing

Reply:
Done, thanks!